scripts/types/globals.d.ts

```diff
@@ -0,0 +1,43 @@
+import { User } from 'auth0'
+
+declare global {
+  /** We inject this into the top of functions, so it will be present */
+  const TEMPLATE_DATA: Record<string, any>
+  /** @tepez/auth0-rules-types declares that the global configuration object is of this type, so we extend it for all uses of configuration, including those in db scripts */
+  interface IAuth0RuleConfiguration {
+    POSTGRES_USERNAME: string
+    POSTGRES_PASSWORD: string
+    POSTGRES_HOST: string
+    POSTGRES_DATABASE: string
+    POSTGRES_PORT?: string
+    AUTH0_CLIENT_ID: string
+    AUTH0_CLIENT_SECRET: string
+  }
+
+  // --- DB Globals ---
+  /** Parfit DB Person */
+  type PersonResult = {
```

Comment:
Seems OK, although I have some worry that we should actually be definining this inline in the script files themselves, because if the query changes, there's no way for the compiler to know that the shape of the result isn't this.